data_collection/gazette/spiders/ce_juazeiro_do_norte.py

```diff
@@ -0,0 +1,52 @@
+import datetime as dt
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class CeJuazeiroDoNorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "2307304"
+    name = "ce_juazeiro_do_norte"
+    allowed_domains = ["juazeiro.ce.gov.br"]
+    start_date = dt.date(year=2009, month=1, day=1)
+    end_date = dt.date.today()
+    base_url = "https://juazeiro.ce.gov.br"
+
+    def start_requests(self):
+        """
+        Creates requests for each date since `start_date`.
+
+        The system allows requests for a date range (from beginning-date to end-date)
+        or for a single date. This spider uses the single date method.
+        """
+        target_date = self.start_date
+        while target_date <= self.end_date:
+            formatted_date = target_date.strftime("%d-%m-%Y")
+            search_url = f"{self.base_url}/Imprensa/Diario-Oficial/{formatted_date}/"
+
+            yield scrapy.Request(
+                url=search_url,
+                method="GET",
```

Comment:
**suggestion**: `scrapy.Request` default method is `GET`, so you don't need to include it here.

Reply:
True, thanks! Removed it.